### PR TITLE
BLOCKED: Fixes #741

### LINKED ISSUE #741: The library is not compatible with webpack
The library is not compatible with webpack

### PR DESCRIPTION
**Pull request by jmdobry**
Removes dynamic `require` statements. Generates an `index.ts` to eagerly load all APIs.

See #741 

BLOCKED: Need to run the generator, but it no longer has access to the code snippet fragments.

### CODE REVIEW

File: apis/acceleratedmobilepageurl/v1.ts [hunk 1]
```diff
@@ -16,7 +16,9 @@
 
 /* jshint maxlen: false */
 
-import {createAPIRequest} from '../../lib/apirequest';
+import {
```
Comment (JustinBeckwith) [inline apis/acceleratedmobilepageurl/v1.ts]: Why did it get formatted like that?  Guessing clang will just put it back after we run `gts fix` against it.

Comment (jmdobry) [reply apis/acceleratedmobilepageurl/v1.ts]: want me to run `gts fix` on this PR?

Comment (JustinBeckwith) [reply apis/acceleratedmobilepageurl/v1.ts]: If you can - guessing that will be very rough

Comment (jmdobry) [reply apis/acceleratedmobilepageurl/v1.ts]: if I have to do it 1 file at a time...yeah...no. Looks like the templates just need to be fixed.

**Pull request merged**